scripts/start/checkFixCfg.R

```diff
@@ -15,7 +15,7 @@ checkFixCfg <- function(cfg, remindPath = ".", testmode = FALSE) {
   refcfg <- gms::readDefaultConfig(remindPath)
   remindextras <- c("backup", "remind_folder", "pathToMagpieReport", "cm_nash_autoconverge_lastrun",
                                "gms$c_expname", "restart_subsequent_runs", "gms$c_GDPpcScen",
-                               "gms$cm_CES_configuration", "gms$c_description", "model")
+                               "gms$cm_CES_configuration", "gms$c_description", "model", "renvLockFromPreceedingRun")
```

Comment:
```suggestion
                               "gms$cm_CES_configuration", "gms$c_description", "model", "renvLockFromPrecedingRun")
```
Although I prefer `previous` anyway :wink: